templates/handbook-clustered.html

```diff
@@ -0,0 +1,117 @@
+    </p>
+
+    <h2 id="book-text">1. Book text</h2>
+
+    <ol>
+        <li>
+            Acquire book text in any form
+            <ul>
+                <li>title</li>
+                <li>subtitle</li>
+                <li>blurp</li>
+                <li>author bio</li>
+            </ul>
+        </li>
+        <li>
+            Typeset book
+            <ul>
+                <li>Affinity Publisher / Adobe InDesign</li>
+                <li>TeX / Tectonic</li>
+                <li>typst</li>
+            </ul>
+        </li>
+    </ol>
+
+    <h2 id="book-design">2. Physical book design</h2>
+
+    <ol>
+        <li>Decide book dimensions</li>
+        <li>Decide paper gsm and calculate spine width</li>
+        <li>
+            Design cover
+            <ul>
+                <li>front</li>
+                <li>back</li>
+                <li>insides</li>
+                <li>spine</li>
+            </ul>
+        </li>
+    </ol>
+
+    <h2 id="printing">3. Printing</h2>
+
+    <ol>
+        <li>Decide number of copies</li>
+        <li>Get quotes from printers</li>
+        <li>Decide retail price</li>
+        <li>
+            ISBN
+            <ul>
+                <li>
+                    Issue and assign ISBN
+                    <ul>
+                        <li>
+                            <a href="https://www.nielsenisbnstore.com/">https://www.nielsenisbnstore.com/</a>
+                        </li>
+                    </ul>
+                </li>
+                <li>
+                    Generate EAN-13 ISBN barcode with zint
+                    <ul>
+                        <li>
+                            <a href="https://en.wikipedia.org/wiki/International_Article_Number">https://en.wikipedia.org/wiki/International_Article_Number</a>
+                        </li>
+                        <li>
+                            <a href="https://github.com/zint/zint">https://github.com/zint/zint</a>
+                        </li>
+                    </ul>
+                </li>
+            </ul>
+        </li>
+    </ol>
+
+    <h2 id="online-presence">4. Online presence</h2>
+
+    <ol>
+        <li>
+            Supply book data
+            <ul>
+                <li>
+                    <a href="https://www.nielsentitleeditor.com/">https://www.nielsentitleeditor.com/</a>
+                </li>
+            </ul>
+        </li>
+        <li>Produce HTML files and create epub</li>
+        <li>
+            Validate epub using epubcheck
+            <ul>
+                <li>
+                    <a href="https://github.com/w3c/epubcheck">https://github.com/w3c/epubcheck</a>
+                </li>
+            </ul>
+        </li>
+        <li>Add book files on website</li>
+    </ol>
+
+</main>
+
+<div class="plug">
+    <div class="plug-text">
+        More details of how this works in practice?
+    </div>
+    <a href="/how-it-works/">How it works</a>
+</div>
+{% endblock content %}
```

Comment:
Could we have a final empty linebreak here? (and generally this is our practice for all our files in the repository!)